common/templates/layouts/base.njk

```diff
@@ -2,49 +2,58 @@
 
 {# GOV.UK Design System Components #}
 {% from "govuk/components/back-link/macro.njk"         import govukBackLink %}
-{% from "govuk/components/button/macro.njk"            import govukButton %}
-{% from "govuk/components/checkboxes/macro.njk"        import govukCheckboxes %}
-{% from "govuk/components/details/macro.njk"           import govukDetails %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
```

Comment:
This feels like your editor autosaving? Would be good to revert these to remove the noise from this set of changes.

If there are style changes we want to make we can make them separately from feature changes.

Reply:
Hopefully sorted